manager/integration/tests/test_snapshot.py

```diff
@@ -726,3 +726,22 @@ def test_snapshot_cr(client, volume_name, settings_reset):  # NOQA
         assert len(snapshots) == 1
         assert snapshots[0].created != created_time
         created_time = snapshots[0].created
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_freeze_file_system_for_snapshot(): # NOQA
+    """
+    1. Record the test start time.
+    2. Set freeze-filesystem-for-snapshot to true.
+    3. Create a Longhorn volume 6 GiB with a replica on every Longhorn node.
+    4. Create a workload pod that runs
+       "dd if=/dev/random of=/path/to/file/on/volume bs=1M count=5000".
+       (Add "status=progress" if running manually to see the progress freeze.)
```

Comment:
Hello @ejweber , does step 4 means to use a workload to mount volume created in step 3 and then write data into mount point?

Reply:
It does. Sorry for any confusion. I changed the wording a bit. Please let me know if it is still unclear.